Jenkinsfile_nightly

```diff
@@ -33,14 +33,6 @@ withNightlyPipeline(type, product, component) {
     ]
   ])
 
-  env.BLOB_ROUTER_URL = 'http://reform-scan-blob-router-aat.service.core-compute-aat.internal'
-  env.PROCESSOR_URL = 'http://bulk-scan-processor-aat.service.core-compute-aat.internal'
-  env.STORAGE_ACCOUNT_URL = 'https://reformscan.aat.platform.hmcts.net'
-  env.IDAM_API_URL = 'https://idam-api.aat.platform.hmcts.net'
-  env.IDAM_CLIENT_REDIRECT_URI = 'https://bulk-scan-orchestrator-aat.service.core-compute-aat.internal/oauth2/callback'
-  env.S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
-  env.CORE_CASE_DATA_API_URL = 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
-
```

Comment:
they are kept there so that we can change it when we run it from Jenkins for e.g point those urls to demo and everything works.

Reply:
at least we will have the provision to change url when we need to

Reply:
ah "configurable manually within jenkins. ok, makes sense